nixos/tests/libuiohook.nix

```diff
@@ -0,0 +1,21 @@
+import ./make-test-python.nix ({ pkgs, lib, ... }: {
+  name = "libuiohook";
+  meta = with lib.maintainers; { maintainers = [ anoa ]; };
+
+  nodes.client = { nodes, ... }:
+      let user = nodes.client.config.users.users.alice;
+      in {
+        imports = [ ./common/user-account.nix ./common/x11.nix ];
+
+        environment.systemPackages = [ pkgs.libuiohook.test ];
+
+        test-support.displayManager.auto.user = user.name;
+      };
```

Comment:
Yeah, that's better fixed in a separate PR. (I don't see any off indentation though, everything looks properly aligned to 2 spaces. Can you point me to what you mean?)

To be honest, I wouldn't make a separate PR for this, since it seems like a very small, non-functionallity-related change; I'd just add it to your next PR relating to the package (like a bump). If you'd like to make a PR for this now though, feel free to request my review on it and I'll merge it ASAP.

Reply:
Ah, it's incredibly minor. Just that lines 6-13 are indented by 4 spaces when they would ideally be 2.

Cleaning it up in the next package bump sounds good to me!